libglusterfs/src/inode.c

```diff
@@ -366,11 +350,13 @@ __inode_ctx_free(inode_t *inode)
                     old_THIS = THIS;
```

Comment:
This can also be moved outside the loop at the beginning. It's expected that many xlators will keep inode context info and implement the _forget_ method, so this will be executed many times.